### PR TITLE
Ak chris/authorisation page design

### PR DESCRIPTION
**Pull request by Akshith-Patil**
**Redesigned Authorisation Page**

## What's new
- New Landing page
- Styled the Login Card
- Redesigned styling for login success & error
- Design works on mobile view

EDIT: Removed background image, and replaced it with a simple linear-gradient transition of green color variants

## Screenshots of New Auth Page
**Login Page**
Sign In Page

**Login Successful**
Successful Login

**Login Error**
Login Error

**Mobile View**
image

### CODE REVIEW

**Review by mt-empty — CHANGES_REQUESTED** [review]
Well done.

I really like the art style, but I'm not sure if we can use it.
But, why is it uploaded twice?

If we are using the green plain one, why is the other one here?

File: pages/signin.tsx [hunk 1]
```diff
@@ -80,28 +81,91 @@ export default function SignIn() {
...
+				</Grid>
+			</Grid>
+
+			<Grid
+				sx={{
+					position: 'fixed',
+					top: '50%',
+					left: {
+						xs: '50%',
+						md: '70.83%' // 5/12 + (1 - 5/12) * 1/2
```
[the remark this answers is not in the document]
Comment (ChristianMarchetta) [reply pages/signin.tsx:134]: They explain how the magic 70.83% is calculated

[the remark this answers is not in the document]
Comment (mt-empty) [reply pages/signin.tsx:134]: Its not clear why these numbers came from, what is 12?

**Review by mt-empty — APPROVED** [review]
This is how it look on a wide screen or any 21 by 9 ratio.
image

I'm not too fussed about it though, only a small percentage of users use widescreens, other than that LGTM, well cone.

Please merge #217 and the resolve conflics

**Pull request merged**